trunk/model/filters/Filter.cs

```diff
@@ -284,7 +284,7 @@ void SaveInternal(XElement e)
 
 		void LoadInternal(XElement e)
 		{
-			options.Load(e);	
+			options = options.Load(e);	

```

Comment:
Sorry for my lazy explanation. I got an LogJoint.Persistence.StorageException and my debugging showed that the options properties are not available outside the Load method any more. Since Options is a struct and structs are passed by value, the extension method needs a ref or it won't alter the outer options. Unfortunately you can not use this and ref at the same time to pass the struct by reference.

Steps to reproduce:
- create a search filter
- export this filter
- reopen logjoint and try to import the exported filter

-> the imported filter won't show up and you get an LogJoint.Persistence.StorageException

Reply:
I see, thanks. Load() should not be an extension method because it's error prone. For now, please use   
```
options = new Search.Options().Load(e);
```

Reply:
Done